com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -651,6 +651,15 @@ public void Dispose()
 
         public void NetworkUpdate(NetworkUpdateStage updateStage)
         {
+            if (m_NetworkManager == null)
+            {
+                Debug.Log("m_NetworkManager is null");
+            }
+            else if (m_NetworkManager.NetworkConfig == null)
+            {
+                Debug.Log("m_NetworkManager.NetworkConfig is null");
+            }
+
             if (!m_NetworkManager.NetworkConfig.UseSnapshotDelta && !m_NetworkManager.NetworkConfig.UseSnapshotSpawn)
```

Comment:
This will throw nullref exception if either of the above are true... should maybe just return after logging above?